fs/layer/node.go

```diff
@@ -62,26 +64,32 @@ func newNode(layerDgst digest.Digest, r reader.Reader, blob remote.Blob) (fusefs
 		return nil, fmt.Errorf("failed to get a TOCEntry of the root")
 	}
 	return &node{
-		r: r,
-		e: root,
-		s: newState(layerDgst, blob),
+		r:        r,
+		e:        root,
+		s:        newState(layerDgst, blob),
+		layerSha: layerDgst,
```

Comment:
Please inherit layerSha to children nodes.
i.e. Please add `layerSha: n.layerSha` to the following:

https://github.com/containerd/stargz-snapshotter/blob/fae5679f3dd22e140c7e404b4fe8b6b5b26b9458/fs/layer/node.go#L170-L175

Reply:
Fixed.